### ISSUE TITLE
timercontrol PlayerDeath

### ISSUE THREAD

**Issue by andrtechno** [polserver/ModernDistro#95]
why at death of the character 1 buff flies, and the others remain.
ContinueOnDeath everywhere 0

it is necessary that at death everything flew. bless, agility, strength etc

**Comment by Admin-Yukiko** [comment]
This was probably not working correctly due to the setting, ContinueOnDeath, being misspelled in timedScripts.cfg. I corrected that and added the setting to all currently implemented timed scripts.

**Comment by andrtechno** [comment]
no, that's not the point

**Comment by andrtechno** [comment]
if death, removes only 1 buff and debuff .. and should, according to the idea, remove all timers?

**Comment by Admin-Yukiko** [comment]
This was still an issue. I think I fixed it in a new commit numbered 7a7acc98a4192baa3fe881eba2fdea80c482ca3b.
Andy, please verify that this is fixed. :)

**Comment by andrtechno** [comment]
timercontrol.src

PlayerDeath function
the problem is that LowerDuration only fires once, then the loop stops..
but if you do it like this:

```
function PlayerDeath(mobile, byref timer_list, byref num_poisons)
	Set_Critical(1);

	foreach timer in ( timer_list )
..................
		//LowerDuration(mobile, event, timer_list, num_poisons); // OLD
		var res := LowerDuration(mobile, event, -1, num_poisons); //NEW
		SleepMS(1);

	endforeach

	Set_Critical(0);

   //if this is not done, then after the resurrection it will not be possible to throw a normal buff again
    EraseObjProperty(mobile,"#TimerControlPID");

	return 1;
endfunction
```
